### PR TITLE
Added info how to update themes to Obsidian 0.13+

### PR DESCRIPTION
**Pull request by chrisgrieser**

### CODE REVIEW

File: 04 - Guides, Workflows, & Courses/Guides/How to update your plugins and CSS for live preview.md [hunk 1]
```diff
@@ -100,6 +100,24 @@ Even better would be to wrap the check in to a helper function such as `isLivePr
 
 ## For Theme Developers
 
+- `.CodeMirror-line` has changed to `.cm-line`
+- active line has a new selector (`.cm-active`), that needs to be added for active line highlighting
+- Cursor styling has a new selector (`.cm-s-obsidian .cm-cursor`)
+- fold markers (the "..." when a a heading / list item is folded) has a new selector (`.cm-foldPlaceholder`)
+- the gutter has a new selector (`.markdown-source-view.mod-cm6 .cm-gutters`)
+- the css class for the search/replace modal (`.document-search-container`) is does not have `position: relative` by default anymore, so you need to add that when the theme moves the location of that modal
```
Comment (kometenstaub) [inline 04 - Guides, Workflows, & Courses/Guides/How to update your plugins and CSS for live preview.md:108]:
```suggestion
- the css class for the search/replace modal (`.document-search-container`) does not have `position: relative` by default anymore, so you need to add that when the theme moves the location of that modal
```

File: 04 - Guides, Workflows, & Courses/Guides/How to update your plugins and CSS for live preview.md [hunk 2]
```diff
@@ -100,6 +100,24 @@ Even better would be to wrap the check in to a helper function such as `isLivePr
...
+- Collapse indicators in Edit Mode are not part of the Code mirror line anymore, meaning they cannot be targeted individually anymore.
+- Live preview being a hybrid of edit and preview mode also also shares certain css classes with *both*, meaning some css targeting elements in preview/edit mode will probably be unintentionally target live preview as well.
```
Comment (kometenstaub) [inline 04 - Guides, Workflows, & Courses/Guides/How to update your plugins and CSS for live preview.md:110]:
```suggestion
- Live preview being a hybrid of edit and reading mode also shares certain css classes with *both*, meaning some css targeting elements in reading/edit mode will probably unintentionally target live preview as well. 
```

File: 04 - Guides, Workflows, & Courses/Guides/How to update your plugins and CSS for live preview.md [hunk 3]
```diff
@@ -100,6 +100,24 @@ Even better would be to wrap the check in to a helper function such as `isLivePr
...
 
+- `.CodeMirror-line` has changed to `.cm-line`
+- active line has a new selector (`.cm-active`), that needs to be added for active line highlighting
+- Cursor styling has a new selector (`.cm-s-obsidian .cm-cursor`)
+- fold markers (the "..." when a a heading / list item is folded) has a new selector (`.cm-foldPlaceholder`)
+- the gutter has a new selector (`.markdown-source-view.mod-cm6 .cm-gutters`)
+- the css class for the search/replace modal (`.document-search-container`) is does not have `position: relative` by default anymore, so you need to add that when the theme moves the location of that modal
+- Collapse indicators in Edit Mode are not part of the Code mirror line anymore, meaning they cannot be targeted individually anymore.
+- Live preview being a hybrid of edit and preview mode also also shares certain css classes with *both*, meaning some css targeting elements in preview/edit mode will probably be unintentionally target live preview as well. 
+	- Example: `.external` is used to style external link in Preview Mode, but is now used in Live preview as well.
```
Comment (kometenstaub) [inline 04 - Guides, Workflows, & Courses/Guides/How to update your plugins and CSS for live preview.md:111]:
```suggestion
	- Example: `.external` is used to style external links in Reading Mode, but is now used in Live preview as well. 
```

File: 04 - Guides, Workflows, & Courses/Guides/How to update your plugins and CSS for live preview.md [hunk 4]
```diff
@@ -100,6 +100,24 @@ Even better would be to wrap the check in to a helper function such as `isLivePr
...
+- active line has a new selector (`.cm-active`), that needs to be added for active line highlighting
+- Cursor styling has a new selector (`.cm-s-obsidian .cm-cursor`)
+- fold markers (the "..." when a a heading / list item is folded) has a new selector (`.cm-foldPlaceholder`)
+- the gutter has a new selector (`.markdown-source-view.mod-cm6 .cm-gutters`)
+- the css class for the search/replace modal (`.document-search-container`) is does not have `position: relative` by default anymore, so you need to add that when the theme moves the location of that modal
+- Collapse indicators in Edit Mode are not part of the Code mirror line anymore, meaning they cannot be targeted individually anymore.
+- Live preview being a hybrid of edit and preview mode also also shares certain css classes with *both*, meaning some css targeting elements in preview/edit mode will probably be unintentionally target live preview as well. 
+	- Example: `.external` is used to style external link in Preview Mode, but is now used in Live preview as well. 
+	- Mostly, these issues can be solved via more specific selectors, e.g. `a.external` for Preview Mode and `span.external` for Edit Mode
```
Comment (kometenstaub) [inline 04 - Guides, Workflows, & Courses/Guides/How to update your plugins and CSS for live preview.md:112]:
```suggestion
	- Mostly, these issues can be solved via more specific selectors, e.g. `a.external` for Reading Mode and `span.external` for Edit Mode
```

File: 04 - Guides, Workflows, & Courses/Guides/How to update your plugins and CSS for live preview.md [hunk 5]
```diff
@@ -100,6 +100,24 @@ Even better would be to wrap the check in to a helper function such as `isLivePr
...
+- trailing whitespace have a CSS class now, allowing them to target them, e.g.
```
Comment (kometenstaub) [inline 04 - Guides, Workflows, & Courses/Guides/How to update your plugins and CSS for live preview.md:113]:
```suggestion
- trailing whitespace have a CSS class now, allowing to target them, e.g. 
```

**Review by kometenstaub — CHANGES_REQUESTED** [review]
I've made some suggestions you can apply directly by clicking `commit suggestion`.

They mostly concern duplicated words and terminology.

**Comment by chrisgrieser** [comment]
thanks, all done!

**Review by kometenstaub — APPROVED** [review]

**Comment by kometenstaub** [comment]
I approved it now. It’s up to you whether @SlRvb should approve as well, I’m not familiar with the css changes.

**Comment by kometenstaub** [comment]
If you merge, maybe squash and merge.

**Comment by chrisgrieser** [comment]
well, the things I collected are certainly accurate, Slv could probably only add some more stuff she found. Which would be another PR, probably. So I'll merge for now.

**Pull request merged**